config/index.js

```diff
@@ -36,7 +36,8 @@ const config = {
     api: {
       bodySizeLimit: process.env.API_BODY_SIZE_LIMIT,
       parameterLimit: process.env.API_PARAMETER_LIMIT,
-      port: process.env.PORT
+      port: process.env.PORT,
+      apiBaseUrl: process.env.API_URL
```

Comment:
this refers to the API you are making and if im not wrong `apiBaseUrl` is for an external api so I dont think both should be inside the `api` property in the config object